buildSrc/src/main/groovy/com/uber/okbuck/experimental/bazel/AndroidLibraryRuleComposer.groovy

```diff
@@ -0,0 +1,28 @@
+package com.uber.okbuck.experimental.bazel
+
+import com.uber.okbuck.composer.AndroidBuckRuleComposer
+import com.uber.okbuck.core.model.AndroidLibTarget
+
+final class AndroidLibraryRuleComposer extends AndroidBuckRuleComposer {
```

Comment:
I would suggest extending the `AndroidBuckRuleComposer` to create a `BazelAndroidBuckRuleComposer` which can then override the definitions of `external` etc. to keep the logic cleaner